modules/apps/portal-workflow/portal-workflow-task-web/src/main/resources/META-INF/resources/edit_workflow_task.jsp

```diff
@@ -191,6 +191,18 @@ renderResponse.setTitle(headerTitle);
 										label="view[action]"
 									/>
 
+									<%
+									PortletURL editURL = assetRenderer.getURLEdit(liferayPortletRequest, liferayPortletResponse);
+									%>
+
+									<c:if test="<%= Validator.isNotNull(editURL) %>">
+										<liferay-frontend:management-bar-button
+											href="<%= assetRenderer.getURLEdit(liferayPortletRequest, liferayPortletResponse).toString() %>"
+											icon="pencil"
+											label="edit"
+										/>
+									</c:if>
+
```

Comment:
@marcosapmf, do we need to add this code? Isn't it already available in some place in the code? After all, the other Portlets using Workflow which allow the asset edit already display the edit icon, right?

Reply:
@natocesarrego right, I am going do check more once :)